ludwig/utils/triton_utils.py

```diff
@@ -19,28 +55,36 @@ def __init__(self, inference_module):
         self.inference_module = inference_module
 
     def forward(self, {input_signature}):
-        inputs: Dict[str, TorchscriptPreprocessingInput] = {input_dict}
-        results = self.inference_module(inputs)
-        return {output_tuple}
+        with torch.no_grad():
+            inputs: Dict[str, {input_type}] = {input_dict}
+            results = self.inference_module.forward(inputs)
+            return {output_tuple}
+"""
+
+FEATURE_RESHAPE_SPEC = """reshape: {{ shape: [ {reshape_dims} ] }}
 """
 
 TRITON_SPEC = """
     {{
-        name: "{prefix}__{index}" # "{name}"
+        name: "{key}" # "{name}"
         data_type: {data_type}
         dims: [ {data_dims} ]
-    }}"""
+        {reshape_spec}}}"""
 
 INSTANCE_SPEC = """
     {{
         count: {count}
         kind: {kind}
     }}"""
 
-TRITON_CONFIG_TEMPLATE = """
-name: "pytorch_raw"
+DYNAMIC_BATCHING_TEMPLATE = """dynamic_batching {
+    max_queue_delay_microseconds: 100
```

Comment:
Can define an export parameters to pass for `max_queue_delay_microseconds`, with default of 100